.github/workflows/deploy-dev.yml

```diff
@@ -2,6 +2,8 @@ name: Dev code deploy
 
 on:
     workflow_dispatch:
+    schedule:
+    - cron:  '0 2 * * *'
```

Comment:
So the dev cron server is on central time, not UTC?

Reply:
Great point, I thought maybe they were (because the old boxes were I think?) but it appears to not be the case
```
aatwood@prod:~$ timedatectl
               Local time: Tue 2020-12-15 22:57:13 UTC
           Universal time: Tue 2020-12-15 22:57:13 UTC
                 RTC time: Tue 2020-12-15 22:57:13
                Time zone: Etc/UTC (UTC, +0000)
System clock synchronized: yes
              NTP service: inactive
          RTC in local TZ: no
```

Reply:
This schedule is for the Github actions runner, which is on UTC: https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#schedule

If you'd prefer a different time, please commit directly to this branch.

Reply:
Done, updated to 0600 UTC